pkg/scheduler/scheduling_application.go

```diff
@@ -703,3 +714,10 @@ func (sa *SchedulingApplication) finishRecovery() {
 			zap.Error(err))
 	}
 }
+
+func createInsufficientQueueResourcesEvent(ask *si.AllocationAsk, message string) (*si.EventRecord, error) {
+	if ask == nil {
+		return nil, fmt.Errorf("could not create Event object from nil AllocationAsk")
+	}
+	return events.CreateRequestEventRecord(ask.AllocationKey, ask.ApplicationID, "InsufficientQueueResources", message)
```

Comment:
Should use `events.CreateRequestEventRecord()` directly, not this wrapper

Reply:
Done